Tests/Packet++Test/Tests/NtpTests.cpp

```diff
@@ -16,7 +16,6 @@
 
 PTF_TEST_CASE(NtpMethodsTests)
 {
-
```

Comment:
nit: do you have an NTP packet example with Kiss-O-Death? If yes we can add it to the tests, otherwise we can merge the PR as is

Reply:
Unfortunately no, I don't have any.

Reply:
ok, we can merge this PR as is 👍